### PR TITLE
feat(ui): confirm status coloring

### PR DESCRIPTION
**Pull request by BillyFigueroa**
## Description
Ensure that all statuses in the various pages have the proper status colors
...

## Related Issue

- #287

### CODE REVIEW

File: ui/src/lib/features/k8s/networks/services/component.test.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import '@testing-library/jest-dom'
 
+import StatusComponent from '$components/k8s/Status/component.svelte'
```
Comment (UncleGedd) [inline ui/src/lib/features/k8s/networks/services/component.test.ts:7]: IMO having a K8s Service be `Active` doesn't make sense. I don't think they can be in any other status, IMO we can remove this column

Comment (BillyFigueroa) [reply ui/src/lib/features/k8s/networks/services/component.test.ts:7]: > IMO having a K8s Service be `Active` doesn't make sense. I don't think they can be in any other status, IMO we can remove this column

Hey Gedd, are you saying we should remove the status column from the table?

Comment (UncleGedd) [reply ui/src/lib/features/k8s/networks/services/component.test.ts:7]: Yes recommending we remove, noting that neither K9s nor Kubewall show a status field

Comment (BillyFigueroa) [reply ui/src/lib/features/k8s/networks/services/component.test.ts:7]: > Yes recommending we remove, noting that neither K9s nor Kubewall show a status field

Copy. Will push a change asap

**Review by UncleGedd — APPROVED** [review]

**Pull request merged**